patches/0001-data-update-appdata-name.patch

```diff
@@ -0,0 +1,32 @@
+diff --git a/data/simple-scan.appdata.xml.in b/data/simple-scan.appdata.xml.in
+index 8a9846bc..3b9b11af 100644
+--- a/data/simple-scan.appdata.xml.in
++++ b/data/simple-scan.appdata.xml.in
+@@ -81,27 +81,6 @@
+         </p>
+       </description>
+     </release>
+-    <release date="2021-02-25" version="40.beta" type="development">
```

Comment:
why remove this version? It's already tagged as "development", so GNOME Software can already filter it out if it's not desired. 3.37.92 is still there too, even if it's in a similar situation.

Reply:
Because appdata validation failed.